x/evm/types/params_test.go

```diff
@@ -47,8 +47,25 @@ func TestParamsValidate(t *testing.T) {
 		},
 		{
 			"invalid chain config",
-			NewParams("ara", true, true, ChainConfig{}, 2929, 1884, 1344),
-			false,
+			NewParams("ara", true, true, ChainConfig{
+				HomesteadBlock: newIntPtr(-1),
+			}, 2929, 1884, 1344),
+			true,
```

Comment:
noticed the original test was wrong so updated it here to properly test its invalid case.